### PR TITLE
Add command to calculate the minimum fee for a transaction

### PR DESCRIPTION
**Pull request by intricate**

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 1]
```diff
@@ -192,6 +203,18 @@ data GenesisCmd
 -- Shelley CLI flag/option data types
 --
 
+newtype ProtocolParamsFile
+  = ProtocolParamsFile FilePath
+  deriving (Show, Eq)
```
Comment (erikd) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:208]: Of you do `deriving newtype` the default `Show` instance will be much nicer.

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 2]
```diff
@@ -961,6 +1005,24 @@ pTxFile fdir =
       <> Opt.help (show fdir ++ " filepath of the Tx.")
       )
 
+pNumTxIns :: Parser NumTxIns
+pNumTxIns =
+  NumTxIns <$>
+    Opt.option Opt.auto
+      (  Opt.long "num-tx-ins"
```
Comment (erikd) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:1012]: Maybe "tx-in-count" instead?

Comment (intricate) [reply cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:1012]: Sure

**Pull request merged**